clients/web/src/models/ApiKeyModel.js

```diff
@@ -22,10 +22,10 @@ var ApiKeyModel = AccessControlledModel.extend({
     save: function () {
         // Scope needs to be sent to the server as JSON
         var scope = this.get('scope');
-        this.attributes.scope = JSON.stringify(scope);
+        this.attributes.scope = JSON.stringify(scope);  // eslint-disable-line backbone/no-model-attributes
```

Comment:
It's better, though ideally we'd have a more seamless serialization mechanism (or really just POST everything as JSON...). I'll change it.